src/pages/signIn/styles/SingIn.ts

```diff
@@ -0,0 +1,14 @@
+export const styleSingIn = {
+    pageWrapper : "bg-[#1F1432] h-[100dvh] relative !px-0 !py-0",
+    countainerBody: "flex flex-col py-6 px-4",
+    logo: "mx-auto mt-[7rem] mb-4 w-[9rem] h-[9rem]",
+    welcome: "font-medium text-2xl mb-2 text-center",
+    description: "font-light text-sm mb-9 text-center",
+    countainerForm: "relative mb-2",
+    input: "border-0  bg-[#1F1432] h-[4rem] px-10 w-full",
```

Comment:
The input background color must be #1C1C27

Reply:
done